src/ert/scheduler/lsf_driver.py

```diff
@@ -87,9 +87,13 @@ class _Stat(BaseModel):
 
 
 class JobData(BaseModel):
-    iens: int
-    job_state: AnyJob
-    submitted_timestamp: float
+    job_id: Optional[str] = None
+    job_state: Optional[AnyJob] = None
+    submitted_timestamp: Optional[float] = None
+    submitted_future: asyncio.Future[bool] = Field(default_factory=asyncio.Future)
```

Comment:
"submitted" is a little bit ambigous, it could be after submit() is called, or when when bsub is called, or when bsub is finished etc. Maybe `submit_function_finished` or something better?

Reply:
Think about how this will be for resubmissions.

If it interferes with that, maybe we need a lock mechanism instead, "submit_is_in_progress" ?